#
# Definition of all trackable objects
# Identifier corresponds to Trackable ID set in Tracking Tools
#
/**:
    ros__parameters:
        rigid_bodies:
            '0':
                pose: Robot_0/pose
                pose2d: Robot_0/ground_pose
                child_frame_id: Robot_0/base_link
                parent_frame_id: world
            '1':
                pose: Robot_1/pose
                pose2d: Robot_1/ground_pose
                child_frame_id: Robot_1/base_link
                parent_frame_id: world
            '2':
                pose: Robot_2/pose
                pose2d: Robot_2/ground_pose
                child_frame_id: Robot_2/base_link
                parent_frame_id: world
            '3':
                pose: Robot_3/pose
                pose2d: Robot_3/ground_pose
                child_frame_id: Robot_3/base_link
                parent_frame_id: world
            '4':
                pose: Robot_4/pose
                pose2d: Robot_4/ground_pose
                child_frame_id: Robot_4/base_link
                parent_frame_id: world

        optitrack_config:
                multicast_address: 239.255.42.99
                command_port: 1510
                data_port: 1511 
